src/objects/BatchedMesh.js

```diff
@@ -832,6 +832,35 @@ class BatchedMesh extends Mesh {
 
 	}
 
+	setGeometryIdAt( instanceId, geometryId ) {
+
+		// return early if the geometry is out of range or not active
+		const drawInfo = this._drawInfo;
+		if ( instanceId >= drawInfo.length || drawInfo[ instanceId ].active === false ) {
+
+			return null;
+
+		}
+
+		drawInfo[ instanceId ].geometryIndex = geometryId;
+
+		return this;
+
+	}
+
+	getGeometryIdAt( instanceId ) {
+
+		const drawInfo = this._drawInfo;
+		if ( instanceId >= drawInfo.length ) {
```

Comment:
We should check if the geometry is "active" here, as well, as we do in other functions.

Reply:
I just add it to the PR to keep this simple. 
But originally I removed it from this PR cause I'm sceptical about this concept of geometry being "active", currently its always active as it cant be deleted from BatchedMesh class Api. Also I'm not convince the concept of a geometry "active" is the right way of removing the geometry.

Reply:
Something like an "active" flag is needed so a geometry can be removed later when the user triggers the compression, though I understand there's no API for this at the moment. See https://github.com/mrdoob/three.js/pull/28638. If you'd like to recommend a different approach I'd suggest making a new issue to discuss it and keep this one focused on additions that are consistent with the current class.